assets/sass/main.scss

```diff
@@ -392,6 +392,46 @@ h6 {
   }
 }
 
+body {
+  width: 100%;
 
+  background-image: linear-gradient(90deg, #004D60, #095627);
+
+  main {
+    width: 100%;
+  }
+}
+
+button.btn-primary {
+  border: 0;
+  border-radius: 5px;
+  padding: 1rem 2.5rem;
+  background-color: #0084BD;
+  color: white;
+  font-size: 1rem;
+}
 
 // Layout
+
+// Sign-in page
+
+#auth_request {
+  margin: 0 auto;
+
+  margin-top: 145px;
```

Comment:
Don't know you're position on using px/em/rem/whatever.  Happy to stick to conventions